set.mm

```diff
@@ -16868,14 +16868,44 @@ either all bound ( ` x , y ` ) or all free ( ` t ` ).  Also note that
       ( wsb weq wi wal sb6 biimpri ) ABCDBCEAFBGABCHI $.
   $}
 
+  ${
+    $d x y $.  $d x ps $.
+    sbievw.is $e |- ( x = y -> ( ph <-> ps ) ) $.
+    $( Version of ~ sbiev with an extra DV condition, not requiring ~ ax-12 .
+       (Contributed by BJ, 18-Jul-2023.) $)
+    sbievw $p |- ( [ y / x ] ph <-> ps ) $=
+      ( wsb weq wi wal sb6 equsalvw bitri ) ACDFCDGAHCIBACDJABCDEKL $.
+  $}
+
+  ${
+    $d y z $.  $d x y $.
+    $( Version of ~ equsb3 with an extra disjoint variable condition on ` x `
+       and ` y ` , not requiring ~ ax-13 .  (Contributed by Raph Levien and FL,
+       4-Dec-2005.)  (Proof shortened by Andrew Salmon, 14-Jun-2011.)  Remove
+       dependency on ~ ax-13 .  (Revised by Wolf Lammen, 19-Jan-2023.) $)
+    equsb3v $p |- ( [ x / y ] y = z <-> x = z ) $=
+      ( weq equequ1 sbievw ) BCDACDBABACEF $.
+  $}
+
   ${
     $d x y $.
     $( Version of ~ equsb1 with a disjoint variable condition, which neither
        requires ~ ax-12 nor ~ ax-13 .  (Contributed by BJ, 11-Sep-2019.)
        Remove dependencies on axioms.  (Revised by Wolf Lammen, 30-May-2023.)
        (Proof shortened by Steven Nguyen, 19-Jun-2023.)  Revise ~ df-sb .
-       (Revised by Steven Nguyen, 11-Jul-2023.) $)
+       (Revised by Steven Nguyen, 11-Jul-2023.)  (Proof shortened by Steven
+       Nguyen, 22-Jul-2023.) $)
```

Comment:
There are three tags very close in time. Maybe you could have only two (adapt as you see fit).